package.json

```diff
@@ -53,7 +53,8 @@
     "scripts": {
         "serve": "vue-cli-service serve",
         "build": "rollup -c",
-        "build-docs": "vue-cli-service build"
+        "build-docs": "vue-cli-service build",
+        "lint": "eslint -c .eslintrc \"src/**/*.js\""
```

Comment:
I believe this should just be `src/` and `.vue` files

i.e: `eslint --ext .js,.vue src/`